pages/Documents/ConversationalAI/Third-PartyBots/e-bot7/introduction.md

```diff
@@ -0,0 +1,70 @@
+---
+pagename: Introduction
+redirect_from:
+  - bot-connectors-ebot7.html
+sitesection: Documents
+categoryname: "Conversational AI"
+documentname: Third-Party Bots
+subfoldername: e-bot7
+permalink: third-party-bots-e-bot7-introduction.html
+indicator:
+---
+
+### Overview
+
+{: .notice}
+Please note the e-bot7 connector is only accessible for early adopters. Enabling the use requires a whitelisting done by
+LivePerson. Contact your LivePerson account representative for assistance.
+
+The following document outlines the configuration of the **e-bot7** connector.
+
+The e-bot7 feature of Third Party Bots allows brands to connect to their existing bots in the ecosystem of e-bot7
+as a service of bot responses inside the Conversational Cloud.
+
+{: .important}
+In order to use this integration you need to make sure to have set-up an account with e-bot7, have an existing bot and created an application to be able to connect with third-party systems. You can read more about applications in the [documentation of e-bot7](https://docs.e-bot7.com/docs/applications/how_apps_work)
```

Comment:
To use this integration you need to make sure to have set up an account with e-bot7, have an existing bot, and created an application to be able to connect with third-party systems. You can read more about applications in the [documentation of e-bot7](https://docs.e-bot7.com/docs/applications/how_apps_work)